Library/Formula/deis.rb

```diff
@@ -1,13 +1,10 @@
 class Deis < Formula
   homepage "http://deis.io"
-  url "https://github.com/deis/deis/archive/v1.4.1.tar.gz"
-  sha1 "4f28c042ae634d92c728b81ea3fed03111b83eb6"
+  url "https://github.com/deis/deis/archive/v1.5.1.tar.gz"
+  sha1 "3a4b0caa0720785a6c093d15a62f2bcbb76324a4"
```

Comment:
Just so you know, we're starting to prefer sha256 over sha1, though this is fine.